dotcom-rendering/src/layouts/CommentLayout.tsx

```diff
@@ -438,9 +439,16 @@ export const CommentLayout = (props: WebProps | AppsProps) => {
 
 			<main data-layout="CommentLayout">
 				{isApps && (
-					<Island priority="critical">
-						<AdPortals />
-					</Island>
+					<>
+						<Island priority="critical">
+							<AdPortals />
+						</Island>
+						<Island priority="critical">
```

Comment:
I think this should be a `feature`. It has no impact on whether or when this loads, but it would be more semantic.